6_visualize.yml

```diff
@@ -48,13 +56,40 @@ targets:
       tmp=I('6_visualize/tmp'),
       log=I('6_visualize/log'))
 
+  6_vizprep/out/view_fun.rds.ind:
+    command: prep_view_fun(target_name, view_polygon)
+  6_vizprep/out/view_fun.rds:
+    command: gd_get('6_vizprep/out/view_fun.rds.ind')
+
+  6_vizprep/out/basemap_fun.rds.ind:
+    command: prep_basemap_fun(target_name, '1_fetch/out/focus_geoms.rds.ind', '1_fetch/out/secondary_geoms.rds.ind')
+  6_vizprep/out/basemap_fun.rds:
+    command: gd_get('6_vizprep/out/basemap_fun.rds.ind')
+
+  6_vizprep/out/storm_line_fun.rds.ind:
+    command: prep_storm_line_fun(target_name, "2_process/out/storm_line.rds.ind")
+  6_vizprep/out/storm_line_fun.rds:
+    command: gd_get('6_vizprep/out/storm_line_fun.rds.ind')
+
+  6_vizprep/out/rivers_fun.rds.ind:
+    command: prep_rivers_fun(target_name, '1_fetch/out/river_geometry.rds.ind')
+  6_vizprep/out/rivers_fun.rds:
+    command: gd_get('6_vizprep/out/rivers_fun.rds.ind')
+
+  6_vizprep/out/watermark_fun.rds.ind:
+    command: prep_watermark_fun(target_name, '6_visualize/in/usgs_logo_black.png')
+  6_vizprep/out/watermark_fun.rds:
+    command: gd_get('6_vizprep/out/watermark_fun.rds.ind')
```

Comment:
Depending on how long these function.rds files take to generate, and how frequently we need to do it, I think we may want to leave these 5 files off of Google Drive (in addition to taking the .png frames off of Drive). We'd be making the decision to not share a cache for these products; instead, the shared cache would take everyone through the fetch and process steps together but anybody who wanted one of these files would need to build it locally. The benefit would be speed of iteration in this 6th phase. Cutting off the shared cache means it's trickier to add a shared-cache file later in the pipeline, i.e., sharing the final .gif as a shared-cache file or sharing other output files that depend on these files...but because we don't have any phases after this one, that truncation of the shared cache shouldn't be a problem for us.

Reply:
I see. Good idea. Can you show me what that looks like to implement?

Reply:
Yes, understood, but I am wondering instead how this works w/ connecting the different remake files. I should've been clear about that. I am in the process of setting these up as .rds and will leave the .yaml connections alone for the time being.

Reply:
Now that I think of it, if we are skipping the cache, is there a reason to use RDS instead of having this function return `plot_fun` and skip the save/readRDS? I am still getting used to when not to use objects vs rds.

Reply:
Hmm, good point about RDS vs objects! I am also still getting used to that. I think you're right that an object would be fine...let's try it as objects, and we can always switch back if needed.

Reply:
Yeah, I think the reasons I want to do that: 1) simplicity, 2) it makes it really clear that it is outside of the scope of gdrive/cache.

Reply:
About connecting the different remake files...I might be misinterpreting, but I think what we want is to (1) cut off the shared cache by switching away from .ind files and toward .rds files or R objects, and (2) keep going with the remake pipeline through phase 6 to the end in the same way as we would for a non-shared-cache pipeline. Therefore:

* This file, 6_visualize.yml, gets the same sort of group target as the 1_fetch.yml and 2_process.yml, but the dependencies are files or objects. Like this:
```yml
target_default: 6_visualize

includes:
  - 2_process.yml

targets:
  6_visualize:
    depends: 
      - basemap_fun
      - watermark_fun
      - [etc]
```
and then the main remake file, `remake.yml`, depends on `6_visualize`:
```yml
target_default: 6_visualize

include:
  - 6_visualize.yml

targets:
[empty]
```
The funny bit comes in at the task plan combine phase, which we'll deal with shortly.

Is that more the type of answer you were looking for?

Reply:
Yes, got it. Makes sense.